content/projects/gsoc/2019/plugin-installation-manager-tool-cli.adoc

```diff
@@ -36,15 +36,22 @@ There are currently many reincarnations of plugin management across Jenkins, inc
 
 This project will create a new Plugin Manager CLI tools and a library which would unify the plugin management across different Jenkins implementations.
 Although we looked at extracting plugin management from Jenkins Core into its own module, this proved to be a challenging first step (see link:https://issues.jenkins-ci.org/browse/JENKINS-57672[JENKINS-57672]).
-Right now the project is focusing on extracting the link:https://github.com/jenkinsci/docker/blob/master/install-plugins.sh[docker install-plugins bash script] to Java.
+Phase 1 focused on extracting the link:https://github.com/jenkinsci/docker/blob/master/install-plugins.sh[docker install-plugins bash script] to Java.
+Phase 2 will include improving upon what was written in Phase 1 and full integration with link:https://github.com/jenkinsci/docker[Jenkins Docker].
 
-== Project Proposal
-You can find more information in the project proposal link:https://docs.google.com/document/d/1lMCDqY5TKVXyFl67BmyMkaS9GTjRbueKr7ds395b_10/edit?usp=sharing[here].
+=== Project Information
+* You can find more information about the original the project proposal link:https://docs.google.com/document/d/1lMCDqY5TKVXyFl67BmyMkaS9GTjRbueKr7ds395b_10/edit?usp=sharing[here].
+* Code can be found link:https://github.com/jenkinsci/plugin-installation-manager-tool[here]
+* Jira tasks and feature requests can be seen link:https://issues.jenkins-ci.org/browse/JENKINS-58199?jql=project%20%3D%20JENKINS%20AND%20component%20%3D%20plugin-installation-manager-tool[here]
 
-== Repository
-Code can be found link:https://github.com/jenkinsci/plugin-installation-manager-tool[here]
+=== Phase 1 Update
+An alpha version of the library and CLI tool has been released! It implements basic functionality of the install-plugins.sh script to take in plugins and download them to a user-specified directory.
```

Comment:
It would be good if the released jar was attached to the release page, you can download it from artifactory and upload it there.